packages/core/src/runtime-metadata.feature.ts

```diff
@@ -1,9 +1,17 @@
 import { Config } from './entities/config';
 import { Feature } from './entities/feature';
 
+export interface FoundFeatures {
+    featureName: string;
+    configurations: string[];
+}
+
 export interface RuntimeMetadataConfig {
     devport?: number;
-    applicationPath: string;
+    applicationPath?: string;
+    isWorkspace?: boolean;
```

Comment:
why is it even relevant whether the source-code of the app sat in a workspace or not? please make sure to avoid mixing build-time concerns with runtime ones.